yetanotherco/aligned_layer
# add feedback to the send proof

This PR introduces feedback for proof verification

To test it send a proof, for example:

```bash
make batcher_send_plonk_bls12_381_task
```
remember to send two tasks

or 

```bash
aligned --proving_system GnarkPlonkBls12_381 \
--proof batcher/aligned/test_files/plonk_bls12_381/plonk.proof \
--public_input batcher/aligned/test_files/plonk_bls12_381/plonk_pub_input.pub \
--vk batcher/aligned/test_files/plonk_bls12_381/plonk.vk \
--repetitions 2
```

You will see a `Waiting for batch signatures verification in Ethereum...` message  every 3 seconds until the proof is verified
